homeassistant/components/bayesian/repairs.py

```diff
@@ -37,3 +37,20 @@ def raise_mirrored_entries(hass: HomeAssistant, observations, text: str = "") ->
             translation_placeholders={"entity": text},
             learn_more_url="https://github.com/home-assistant/core/pull/67631",
         )
+
+
+# Should deprecate in some future version (2022.10 at time of writing) & make prob_given_false required in schemas.
+def raise_no_prob_given_false(hass: HomeAssistant, observation, text: str) -> None:
+    """In previous 2022.9 and earlier, prob_given_false was optional and had a default version."""
+    issue_registry.async_create_issue(
+        hass,
+        DOMAIN,
+        f"no_prob_given_false/{text}",
+        breaks_in_ha_version="2022.10.0",
+        is_fixable=False,
+        is_persistent=False,
```

Comment:
`is_persistent` is False by default.

Reply:
Ok - fixed in.
I like declaring default values as it helps me read and adjust code in future. But I am an amateur and want to adhere to guidelines. 
Fixed in [d895c7a](https://github.com/home-assistant/core/pull/79303/commits/d895c7ab3100916cf41281b1ac001094ad0d2ac2)